test/functional/aggregation_tests.js

```diff
@@ -540,6 +540,9 @@ describe('Aggregation', function() {
             expect(result._id.tags).to.equal('good');
             expect(result.authors).to.eql(['bob']);
 
+            // Since cursor will not be "exhausted", since batchSize is 1,
```

Comment:
I think we should endeavor to clean up non-exhausted cursors when the parent client is closed - I don't think this should be our policy, or something we're educating our users to do explicitly, it's too error prone in the real world

Reply:
Does this mean we are tracking all cursors in the client?

Reply:
discussed offline, this is fine because we are explicitly testing this